medical_medication/tests/test_medical_medication_template.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp.tests.common import TransactionCase
+
+
+class TestMedicalMedicationTemplate(TransactionCase):
+
+    def setUp(self):
+        super(TestMedicalMedicationTemplate, self).setUp()
+        self.med_template_1 = self.env.ref(
+            'medical_medication.medical_medication_template_template_1'
+        )
+
+        self.med_template_2 = self.env.ref(
+            'medical_medication.medical_medication_template_template_2'
+        )
+
+        self.med_template_3 = self.env.ref(
+            'medical_medication.medical_medication_template_template_3'
+        )
+
+        self.med_template_4 = self.env.ref(
+            'medical_medication.medical_medication_template_template_4'
+        )
+
+    def test_name_get_dosage_id(self):
+        """ Test display_name if dosage_id name chosen """
+        self.assertEquals(
+            self.med_template_4.display_name, 'as needed',
+            'Should show dosage_id name.\rGot: %s\rExpected: %s' % (
```

Comment:
No need to remove, but these strings are superfluous - the default fail says basically the same thing. If you name your methods well and have a descriptive first line of the docblock, you're good with just the default messages IMO.